<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="sariful" failures="1" tests="2" name="Test(failed)(failed)" time="32.505" errors="0" timestamp="2024-08-20T15:11:25 EDT">
  <testcase classname="codenbox.testCases.LoginTest" name="verify_login" time="11.488">
    <skipped/>
  </testcase> <!-- verify_login -->
  <testcase classname="codenbox.testCases.LoginTest" name="verify_login" time="11.407">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at codenbox.testCases.LoginTest.verify_login(LoginTest.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- Test(failed)(failed) -->
